csharp/src/Google.Protobuf/Reflection/EnumDescriptor.cs

```diff
@@ -68,6 +68,14 @@ internal EnumDescriptor(EnumDescriptorProto proto, FileDescriptor file, MessageD
 
         internal EnumDescriptorProto Proto { get { return proto; } }
 
+        /// <summary>
+        /// Returns a clone of the underlying <see cref="EnumDescriptorProto"/> describing this enum.
+        /// Note that a copy is taken every time this method is called, so clients using it frequently
```

Comment:
nit: worth mentioning that a deep clone is taken? (which also explains why it's safe to to .Clone() internally since it provides a deep clone).
Same below.
Leaving resolution up to you - I don't feel strongly about this at all.